### PR TITLE
Updated README with new links and fix almost all markdown lint errors

### PR DESCRIPTION
**Pull request by julian-alarcon**
* Updated links for github actions badge
* Renamed some main titles from gmusic to ytmusic
* Fixed almost all CommonMark lint errors https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md

Pending: As documention mentions a lot gmusic it needs to be updated according to the new code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,79 +58,96 @@ window.gmusic.volume.getVolume(); // 50 (ranges from 0 to 100)
 ```
 
 ## Documentation
-`gmusic.js` exposes a constructor, `GMusic` as its `module.exports` (`window.GMusic` for `bower`/vanilla).
+
+`gmusic.js` exposes a constructor, `GMusic` as its `module.exports` (`window.GMusic` for
```
Comment (jostrander) [inline README.md]: Should this be changed to `ytmusic.js` as well?

File: README.md [hunk 2]
```diff
@@ -395,25 +461,30 @@ npm test
 # npm run test-windows
 ```
 
-##### Attribution
-We have an automatic build setup on Travis CI that will run every day to catch changes made by Google as quick as possible. These tests make use of BrowserStack's services to remotely run Selenium tests, huge thanks to [BrowserStack][] for the support of Open Source projects like this one!
+#### Attribution
```
Comment (jostrander) [inline README.md]: While were here, this should probably be removed.  We no longer test nightly as we don't have any of the infrastructure set up to do so.

**Review by jostrander — CHANGES_REQUESTED** [review]
Just a few small changes and I'll merge this in.

File: README.md (hunk 2 above)
Comment (julian-alarcon) [reply README.md]: Working on it

**Comment by julian-alarcon** [comment]
Can you check again @jostrander ?

**Review by jostrander — APPROVED** [review]

**Pull request merged**